### DISCUSSION TITLE
2 weeks after we moved to Github, our organization got flagged. That was over two months ago with no word

### DISCUSSION THREAD

**Discussion by Teyke** [community/community#121486] [New to GitHub]
My account also got flagged twice (apparently by mistake both times; each time it was unfrozen without me having to take any further action than fill out an incredibly annoying number of captchas). But the organization that my volunteering group created for our project continues to be flagged since February 19th. One of my co-workers filed a ticket at the time; I filed one a month later (#2710618). No word on either. We have no idea what could possibly be the problem with the organization and they won't tell us, or say anything at all. 

Is there something in particular that we need to do to get an organization unfrozen? We're all pretty new to Github so maybe we're just missing something, but this seems really excessive. If there's a particular settings or things that need to be filled out on the organization's profile, we'd happily do it if only we knew what it was.

**Comment by ghostinhershell** [comment]
Hi @Teyke,  

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
